docs/core/porting/breaking-changes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Breaking changes
+description: Breaking changes between releases of .NET and how it impacts compatibility.
+author: stebon
+ms.date: 06/14/2021
+---
+# Breaking Changes
```

Comment:
This is probably too generic and conflicts with the existing content. We don't want the search engines or users confused. I'm having a hard time figuring out what to put here that differentiates this article from others related to breaking changes. Please feel free to change this.

- Title changed
- Description needs to be longer
- `ms.author` is your alias
- `author` is your github id

```suggestion
---
title: Breaking changes can affect porting your app
description: Breaking changes can occur when porting your code from .NET framework to .NET. This article describes the categories of breaking changes you may encounter when porting.
author: StephenBonikowsky
ms.author: stebon
ms.date: 06/14/2021
---
# Breaking changes may occur when porting code
```